janitor/utils.py

```diff
@@ -231,6 +226,53 @@ def idempotent(func: Callable, df: pd.DataFrame, *args, **kwargs):
         )
 
 
+def deprecated_kwargs(
+    *arguments: list[str],
+    message: str = (
+        "The keyword argument '{argument}' of '{func_name}' " "is deprecated."
```

Comment:
There's a duplicated `"` in the middle of the default argument. Understand that it is resulted from running `black`, can you remove it?